infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -163,10 +152,26 @@ cmake_llvm $CMAKE_EXTRA_ARGS \
     -DLLVM_USE_SANITIZER=DataFlow \
     -DCMAKE_INSTALL_PREFIX=/usr/dfsan/
 
-ninja -j $NPROC cxx cxxabi
-ninja install-cxx install-cxxabi
+ninja -j $NPROC install-cxx install-cxxabi
 rm -rf $WORK/dfsan
 
+
+# AARCH64
+mkdir $WORK/aarch64-cxx
+cd $WORK/aarch64-cxx
+cmake -DLIBCXX_ENABLE_SHARED=OFF -DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=ON -DLIBCXXABI_ENABLE_SHARED=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="$AARCH64_EXTRA_CFLAGS" -DCMAKE_C_COMPILER=clang -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=lld" -DCMAKE_C_COMPILER_TARGET=$AARCH64_TARGET_TRIPLE -DCMAKE_ASM_COMPILER_TARGET=$AARCH64_TARGET_TRIPLE -DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN=$AARCH64_TOOLCHAIN -DCMAKE_SYSROOT=$AARCH64_SYSROOT -DCMAKE_INSTALL_PREFIX=$AARCH64_SYSROOT -DLLVM_ENABLE_PROJECTS="libcxx;libcxxabi" $LLVM_SRC/llvm
```

Comment:
can you use the existing `cmake_llvm` function with additional ARM-specific arguments passed to it? Otherwise we're duplicating many flags

Reply:
if you need to land this fast to test more things out, please add a TODO that cmake calls here and on line 168 should be unified